README.md

```diff
@@ -22,6 +22,38 @@ Update config.yml and run
 cp2influxdb -c config.yml
```

Comment:
Use `mdcat` format - will make your life easier, and sharing HARDWARIO's configs more consistent :)

https://pypi.org/project/mdcat/